ci-operator/config/openshift/lightspeed-service/openshift-lightspeed-service-main.yaml

```diff
@@ -4,8 +4,8 @@ build_root:
     namespace: openshift
     tag: golang-1.13
```

Comment:
this is missing the actual image build definition stanza, see https://github.com/openshift/release/blob/bd75d440f21d0652f9ec28b9d2abc0a3abbd4299/ci-operator/config/openshift/cluster-image-registry-operator/openshift-cluster-image-registry-operator-master.yaml#L20-L27 for an example.

I think that addresses your question about the container file.

https://docs.ci.openshift.org/docs/architecture/images/ has some related info

Reply:
Thanks for sharing. Referred and added snippets, verifying the results in rehearsal jobs.